types/creativebulma__bulma-tagsinput/creativebulma__bulma-tagsinput-tests.ts

```diff
@@ -0,0 +1,143 @@
+import BulmaTagsInput, { BulmaTagsInputItem } from 'creativebulma__bulma-tagsinput';
```

Comment:
this should be change after paths are corrected to:
```suggestion
import BulmaTagsInput, { BulmaTagsInputItem } from '@creativebulma/bulma-tagsinput';
```